app/src/main/java/pomobox/ui/break_tasks/fragment/BreakTasksFragment.java

```diff
@@ -0,0 +1,149 @@
+package pomobox.ui.break_tasks.fragment;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Build;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.SeekBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+import pomobox.R;
+import pomobox.base.BaseCustomDialog;
+import pomobox.base.BaseFragment;
+import pomobox.data.model.BreakTask;
+import pomobox.data.database.BreakTaskHelperDB;
+import pomobox.ui.break_tasks.holder.BreakTaskAdapter;
+
+public class BreakTasksFragment extends BaseFragment implements BreakTaskContract.View {
+
+    private BreakTaskHelperDB mHelper;
+    private RecyclerView mRecyclerBreakTask;
+    private ArrayList<BreakTask> mTaskDataList;
+    private BreakTaskAdapter mAdapter;
+    private Context mContext;
+    private ImageButton mButtonAdd;
+    private BreakTaskPresenter mTaskPresenter;
+    public static BreakTasksFragment newInstance() {
+        return new BreakTasksFragment();
+    }
+
+    @Override
+    public int getLayoutId() {
+        return R.layout.fragment_break_tasks;
+    }
+
+    @Override
+    public void onViewReady(View view) {
+        mContext = getContext();
+        mRecyclerBreakTask = view.findViewById(R.id.recycler_break_tasks);
+        mButtonAdd = view.findViewById(R.id.button_add_list);
+        mHelper = new BreakTaskHelperDB(mContext);
+        mTaskDataList = mHelper.getAllTasks();
+        mTaskPresenter = new BreakTaskPresenter(this, mHelper);
+        doRecyclerView();
+        setButtonAdd();
+    }
+
+    private void doRecyclerView(){
```

Comment:
"do" chỉ đi kèm với những công việc học tập, việc nhà, chăm sóc bản thân thôi. còn lại dùng "make" 
https://www.espressoenglish.net/difference-between-do-and-make-60-collocations/
```suggestion
    private void initRecyclerView(){
```